nextjs/app/page.tsx

```diff
@@ -0,0 +1,27 @@
+import Better from "@/Components/MainPage/Better";
+import Copilot from "@/Components/MainPage/Copilot";
+import Detail from "@/Components/MainPage/Detail";
+import Feature from "@/Components/MainPage/Feature";
+import Main1 from "@/Components/MainPage/Main1";
+import More from "@/Components/MainPage/More";
+import Product from "@/Components/MainPage/Product";
+import ProductImg from "@/Components/MainPage/ProductImg";
+import Promote from "@/Components/MainPage/Promote";
+import Video from "@/Components/MainPage/Video";
+
+export default function Home() {
+  return (
+    <>
+      <Main1></Main1>
+      <Promote></Promote>
+      <Feature></Feature>
+      <Copilot></Copilot>
+      <Video></Video>
+      <Product></Product>
+      <ProductImg></ProductImg>
+      <Better></Better>
+      <More></More>
+      <Detail></Detail>
```

Comment:
if the component doesn't wrap anything you should do it as a single tag, e.g.:
` <Main1 />`

Reply:
Everything has been fixed. Thank you for your comments!!